.github/workflows/release-images.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
     name: Release on GitHub
     runs-on: ubuntu-20.04
     container:
-      image: 1.16.2.1_focal
+      image: golang:1.16.2.1_focal
```

Comment:
Did you mean?
```suggestion
      image: quay.io/cybozu/golang:1.16-focal
```

Reply:
Thank you for pointing out. Yes that is a mistake.
However, by checking this job in my repo, specifying the quay image in the field will not work because the account is not logged in as a quay user. 

```
Checking docker version
Clean up resources from previous jobs
Create local container network
Starting job container
  /usr/bin/docker pull quay.io/cybozu-go/golang:1.16-focal
  Error response from daemon: unauthorized: access to the requested resource is not authorized
  Warning: Docker pull failed with exit code 1, back off 5.233 seconds before retry.
  /usr/bin/docker pull quay.io/cybozu-go/golang:1.16-focal
  Error response from daemon: unauthorized: access to the requested resource is not authorized
  Warning: Docker pull failed with exit code 1, back off 2.296 seconds before retry.
  /usr/bin/docker pull quay.io/cybozu-go/golang:1.16-focal
  Error response from daemon: unauthorized: access to the requested resource is not authorized
  Error: Docker pull failed with exit code 1
```

The reason why we are trying to use the image is just that that already contains `ghr`.
But as using the image as a base is impossible, it is required to download and use it on the VM.

Reply:
Thank you. Yes that must be `quay.io/cybozu/golang:1.16-focal` and this would work.